package.json

```diff
@@ -50,16 +50,18 @@
     "eslint": "6.8.0",
     "jest": "29.4.2",
     "lint-staged": "13.1.2",
+    "memory-fs": "0.5.0",
     "prettier": "2.8.4",
     "typescript": "4.9.5"
   },
   "scripts": {
     "build": "npm run build:types && npm run build:js",
```

Comment:
"pre-commit": "npm run test",

Reply:
In my experience, running all tests on each commit is bad DX.

I could entertain the idea of running them before pushing, but a much better option imo (more reliable, better DX) would be to set up CI on this repository. I have created #19 to track this.

Reply:
It's covered in separate [PR](https://github.com/xlts-dev/verdaccio-google-cloud-storage/pull/20)